stylesheets/base/colors.css

```diff
@@ -0,0 +1,8 @@
+:root {
+  --primary-color: hsl(235, 86%, 65%);
+  --primary-hover: hsl(235, 87%, 70%);
+  --secondary-color: hsl(248, 10%, 30%);
+  --secondary-color:hsl(248, 10%, 40%);
```

Comment:
I think this should have been --secondary-hover instead of --secondary-color.

Reply:
Oh yes, it was supposed to be that. Thank you for pointing out